### DISCUSSION TITLE
Sponcorship?

### DISCUSSION THREAD

**Discussion by exzachlyvv** [SeaQL/sea-orm#299] [Ideas]
I would love to contribute financially to this project or the SeaQL organization. Have you considered turning that on? Maybe I missed it?

**Comment by billy1624** [comment]
OMG, our first sponsor! Huge thanks! We are setting things up for GitHub sponsors now.

**Comment by tyt2y3** [comment]
Thank you so much for the kind words! It really will be a big motivation for us. We are still at an early stage, and I hope we'd provide enough value to the community to shamelessly 'call for donations' soon.

I am trying to setup GitHub sponsor now, stay tuned )

**Reply by tyt2y3** [reply]
![image](https://user-images.githubusercontent.com/1782664/140881477-69652e97-5a60-4590-829f-f9c5ca042076.png)

Submitted for approval yesterday, hopefully will go live soon.

**Reply by exzachlyvv** [reply]
Cool! No rush. Just update me once approved :D

**Reply by billy1624** [reply]
Hey @exzachlyvv, GitHub Sponsor is now available! :P
https://github.com/sponsors/SeaQL

**Reply by exzachlyvv** [reply]
😁

**Reply by tyt2y3** [reply]
Thank you so much for the support!

As a courtesy we added an acknowledgement to our blog post https://www.sea-ql.org/SeaORM/blog/2021-11-19-whats-new-in-0.4.0/#sponsor
